src/components/playground/playground.scss

```diff
@@ -195,13 +191,6 @@ section.example {
     background-color: rgb(var(--kompendium-contrast-100));
     border: 1px solid rgb(var(--kompendium-contrast-300));
     box-shadow: var(--shadow-showcase);
```

Comment:
:zap:

This was just a _tiny_ bit more complicated than it looks, because of shadow-DOM… 😝

[`0384e01` (#57)](https://github.com/jgroth/kompendium/pull/57/commits/0384e010dc77545e84d0ef905533bd755e8dc666)